content/python/concepts/errors/errors.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Errors'
-Description: 'There are (at least) two distinguishable kinds of errors in Python: syntax errors and exceptions. Syntax errors, also known as parsing errors, are perhaps the most common kind of complaint you get while you are still learning Python: error File "script.py", line 1 while True print("Hello world!") ^ SyntaxError: invalid syntax'
+Description: 'There are two main kinds of errors in Python: syntax errors and exceptions. Exceptions are any errors that arise from syntactically correct code.
```

Comment:
```suggestion
Description: 'There are two main kinds of errors in Python: syntax errors and exceptions. Exceptions are any errors that arise from syntactically correct code.'
```

Noticed this was missing a closing single-quote. This suggestion should fix the one GitHub (test) check that keeps failing. 😄